app/views/proposal/show.html.haml

```diff
@@ -47,6 +47,15 @@
       .row
         .col-md-12
           %p
+            - if @event.state == 'confirmed' && !@event.intersecting_events.withdrawn.empty?
```

Comment:
This is not at all visible here. I wonder how we could make it more obvious to visitors.